src/main/java/seedu/address/model/internship/InternshipApplication.java

```diff
@@ -26,6 +26,8 @@
     private final Status status;
     private final ArrayList<Interview> interviews;
     private Boolean isArchived;
+    private Boolean isGhostedOrRejected;
+    private String lastStage;
```

Comment:
Maybe the lastStage should be of type Status instead? Or perhaps there is a reason for it to be string?

Reply:
I put it as a String because to make it such that the message to notify users of the last stage failed only appears when the status is changed to ghosted/rejected else it does not, my current implementation is for lastStage to be an empty string so it does not appear (the default is empty string unless set)

Reply:
Is it not possible to just use Optional<Status> and set the default to Optional.null?

Reply:
Okay! I will see how I can change to Status later!